modules/ossm-rn-known-issues.adoc

```diff
@@ -21,21 +21,34 @@ These limitations exist in {SMProductName}:
 
 * The first time you access related services such as {JaegerShortName} and Grafana, from the Kiali console, you must accept the certificate and re-authenticate using your {product-title} login credentials. This happens due to an issue with how the framework displays embedded pages in the console.
 
-* The Bookinfo sample application cannot be installed on IBM Z and IBM Power Systems.
+* The Bookinfo sample application cannot be installed on IBM Z and IBM Power.
 
-* WebAssembly extensions are not supported on IBM Z and IBM Power Systems.
+* WebAssembly extensions are not supported on IBM Z and IBM Power.
 
+* LuaJIT is not supported on IBM Power.
 
 [id="ossm-rn-known-issues-ossm_{context}"]
 == {SMProductShortName} known issues
 
 These are the known issues in {SMProductName}:
 
+* https://issues.redhat.com/browse/OSSM-1668[OSSM-1668]
+`jwksResolverCA` field is missing in `SMCP`.
+
+* https://issues.redhat.com/browse/OSSM-1655[OSSM-1655]
+Kiali dashboard shows error after enabling mTLS in `SMCP`.
+
 * https://issues.redhat.com/browse/OSSM-1211[OSSM-1211]
 Configuring Federated service meshes for failover does not work as expected. There is no workaround at this time.
 
 * link:https://github.com/istio/istio/issues/14743[Istio-14743] Due to limitations in the version of Istio that this release of {SMProductName} is based on, there are several applications that are currently incompatible with {SMProductShortName}. See the linked community issue for details.
 
+* https://issues.redhat.com/browse/OSSM-1396[OSSM-1396] If gateway service contains `externalIPs` and the `ServiceMeshControlPlane` is updated then the service is removed.
+
+* https://issues.redhat.com/browse/OSSM-1168[OSSM-1168] Service Mesh sidecar injection not working reliably.
+
+* https://issues.redhat.com/browse/OSSM-1052[OSSM-1052] Gateway service template is missing `ExternalIP` parameters.
+
```

Comment:
Same issue as above, we need to provide more information than just the heading from the issues.

Reply:
OSSM-1396 "If a gateway resource contains the spec.externalIPs setting, instead of being recreated when the ServiceMeshControlPlane is updated, the gateway is removed and never recreated."

OSSM-1168 "When service mesh resources are created as a single YAML file, the Envoy proxy sidecar is not reliably injected into pods.  When the SMCP, SMMR, and Deployment resources are created individually, the deployment works as expected."

OSSM-1052 "When configuring a Service ExternalIP for the ingressgateway in the service mesh control plane, the service is not created. The schema for the SMCP is missing the parameter for the service.  The workaround for this issue is to disable the gateway creation in the SMCP spec and manage the gateway deployment entirely manually (including Service, Role and RoleBinding).